2019-06-21 17:09:20 ERROR ControllerService:45 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:40)
	at com.revature.controller.Controller.launch(Controller.java:47)
	at com.revature.Main.main(Main.java:18)
2019-06-21 17:11:22 ERROR ControllerService:45 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:40)
	at com.revature.controller.Controller.launch(Controller.java:47)
	at com.revature.Main.main(Main.java:18)
2019-06-22 05:32:20 ERROR ControllerService:45 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:40)
	at com.revature.controller.Controller.<init>(Controller.java:46)
	at com.revature.Main.main(Main.java:17)
2019-06-22 05:37:38 ERROR ControllerService:45 - Could not login.
2019-06-22 06:07:43 ERROR ControllerService:51 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:44)
	at com.revature.controller.Controller.<init>(Controller.java:40)
	at com.revature.Main.main(Main.java:17)
2019-06-22 06:13:02 ERROR ControllerService:50 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:43)
	at com.revature.controller.Controller.<init>(Controller.java:40)
	at com.revature.Main.main(Main.java:17)
2019-06-23 19:57:36 ERROR ControllerService:50 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:43)
	at com.revature.controller.Controller.<init>(Controller.java:40)
	at com.revature.Main.main(Main.java:17)
2019-06-23 20:27:46 ERROR ControllerService:50 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:43)
	at com.revature.controller.Controller.<init>(Controller.java:40)
	at com.revature.Main.main(Main.java:17)
2019-06-24 01:47:40 ERROR ControllerService:55 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:48)
	at com.revature.controller.Controller.<init>(Controller.java:40)
	at com.revature.Main.main(Main.java:17)
2019-06-24 02:25:18 ERROR ControllerService:126 - Could not set balance.
java.sql.SQLException: Could not commit with auto-commit set on
	at oracle.jdbc.driver.PhysicalConnection.commit(PhysicalConnection.java:4443)
	at oracle.jdbc.driver.PhysicalConnection.commit(PhysicalConnection.java:4490)
	at com.revature.service.ControllerService.depositMoney(ControllerService.java:123)
	at com.revature.controller.Controller.<init>(Controller.java:51)
	at com.revature.Main.main(Main.java:17)
2019-06-24 02:28:51 ERROR ControllerService:127 - Could not set balance.
java.sql.SQLException: Could not commit with auto-commit set on
	at oracle.jdbc.driver.PhysicalConnection.commit(PhysicalConnection.java:4443)
	at oracle.jdbc.driver.PhysicalConnection.commit(PhysicalConnection.java:4490)
	at com.revature.service.ControllerService.depositMoney(ControllerService.java:124)
	at com.revature.controller.Controller.<init>(Controller.java:51)
	at com.revature.Main.main(Main.java:17)
2019-06-24 15:16:17 ERROR ControllerService:56 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:49)
	at com.revature.controller.Controller.<init>(Controller.java:40)
	at com.revature.Main.main(Main.java:17)
2019-06-24 16:10:19 ERROR ControllerService:56 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:49)
	at com.revature.junitTests.ControllerTest.loginTest(ControllerTest.java:55)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:239)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:541)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:763)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:463)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:209)
2019-06-24 16:11:15 ERROR ControllerService:56 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:49)
	at com.revature.junitTests.ControllerTest.loginTest(ControllerTest.java:55)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:239)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:541)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:763)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:463)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:209)
2019-06-24 16:13:23 ERROR ControllerService:56 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:49)
	at com.revature.junitTests.ControllerTest.loginTest(ControllerTest.java:55)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:239)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:541)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:763)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:463)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:209)
2019-06-24 17:15:49 ERROR ControllerService:56 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:49)
	at com.revature.junitTests.ControllerTest.loginTest(ControllerTest.java:52)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:239)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:541)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:763)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:463)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:209)
2019-06-25 09:58:14 ERROR ControllerService:56 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:49)
	at com.revature.controller.Controller.<init>(Controller.java:40)
	at com.revature.Main.main(Main.java:17)
2019-06-25 10:32:48 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER, p4ssw0rd
2019-06-25 10:34:58 TRACE ControllerService:48 - Entering login method with parameters: ljdlkasj, asljlkjd
2019-06-25 10:35:00 ERROR ControllerService:56 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:49)
	at com.revature.controller.Controller.<init>(Controller.java:40)
	at com.revature.Main.main(Main.java:17)
2019-06-25 10:36:13 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER, p4ssw0rd
2019-06-25 12:36:52 TRACE ControllerService:24 - Instantiation has been restricted.  REMOVE THIS LINE BEFORE FINALIZATION!
2019-06-25 12:37:51 TRACE ControllerService:48 - Entering login method with parameters: saljlskfj, aljlsfj
2019-06-25 12:37:52 ERROR ControllerService:56 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:49)
	at com.revature.controller.Controller.<init>(Controller.java:40)
	at com.revature.Main.main(Main.java:17)
2019-06-25 12:38:11 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER, p4ssw0rd
2019-06-25 12:40:19 TRACE ControllerService:116 - Entering deposit method with parameters: balance = 150.0, deposit = 33.29
2019-06-25 12:40:19 TRACE ControllerService:118 - Getting connection with user parameters: TEST_USER, p4ssw0rd
2019-06-25 12:40:19 INFO  ControllerService:124 - REMOVE BEFORE FINALIZATION! Amount to be updated to: 183.29
2019-06-25 12:41:31 TRACE ControllerService:162 - Entering withdraw method with parameters: balance = 183.29, withdraw = 2.05
2019-06-25 12:41:31 TRACE ControllerService:164 - Getting connection with user parameters: TEST_USER, p4ssw0rd
2019-06-25 12:41:31 INFO  ControllerService:170 - REMOVE BEFORE FINALIZATION! Amount to be updated to: 181.23999999999998
2019-06-25 13:32:43 TRACE ControllerService:24 - Instantiation has been restricted.  REMOVE THIS LINE BEFORE FINALIZATION!
2019-06-25 13:33:09 TRACE ControllerService:48 - Entering login method with parameters: dlkjskf, aljdlj
2019-06-25 13:33:11 ERROR ControllerService:56 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:49)
	at com.revature.controller.Controller.<init>(Controller.java:40)
	at com.revature.Main.main(Main.java:17)
2019-06-25 13:37:53 TRACE ControllerService:24 - Instantiation has been restricted.  REMOVE THIS LINE BEFORE FINALIZATION!
2019-06-25 13:38:08 TRACE ControllerService:48 - Entering login method with parameters: kljsd, sljlfs
2019-06-25 13:38:09 ERROR ControllerService:56 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:49)
	at com.revature.controller.Controller.<init>(Controller.java:40)
	at com.revature.Main.main(Main.java:17)
2019-06-25 13:43:57 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER, p4ssw0rd
2019-06-25 13:49:46 TRACE ControllerService:24 - Instantiation has been restricted.  REMOVE THIS LINE BEFORE FINALIZATION!
2019-06-25 13:50:21 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER, p4ssw0rd
2019-06-25 13:51:19 TRACE ControllerService:116 - Entering deposit method with parameters: balance = 0.0, deposit = 50.0
2019-06-25 13:51:19 TRACE ControllerService:118 - Getting connection with user parameters: TEST_USER, p4ssw0rd
2019-06-25 13:51:19 INFO  ControllerService:124 - REMOVE BEFORE FINALIZATION! Amount to be updated to: 50.0
2019-06-25 13:53:05 TRACE ControllerService:24 - Instantiation has been restricted.  REMOVE THIS LINE BEFORE FINALIZATION!
2019-06-25 13:53:18 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER, p4ssw0rd
2019-06-25 13:53:37 TRACE ControllerService:116 - Entering deposit method with parameters: balance = 50.0, deposit = 50.0
2019-06-25 13:53:37 TRACE ControllerService:118 - Getting connection with user parameters: TEST_USER, p4ssw0rd
2019-06-25 13:53:37 INFO  ControllerService:124 - REMOVE BEFORE FINALIZATION! Amount to be updated to: 100.0
2019-06-25 13:53:58 TRACE ControllerService:164 - Entering withdraw method with parameters: balance = 100.0, withdraw = 50.0
2019-06-25 13:53:58 TRACE ControllerService:166 - Getting connection with user parameters: TEST_USER, p4ssw0rd
2019-06-25 13:53:58 INFO  ControllerService:172 - REMOVE BEFORE FINALIZATION! Amount to be updated to: 50.0
2019-06-25 14:09:09 TRACE ControllerService:24 - Instantiation has been restricted.  REMOVE THIS LINE BEFORE FINALIZATION!
2019-06-25 14:09:15 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER, p4ssw0rd
2019-06-25 14:09:35 TRACE ControllerService:116 - Entering deposit method with parameters: balance = 50.0, deposit = 0.0
2019-06-25 14:09:35 TRACE ControllerService:118 - Getting connection with user parameters: TEST_USER, p4ssw0rd
2019-06-25 14:09:35 INFO  ControllerService:124 - REMOVE BEFORE FINALIZATION! Amount to be updated to: 50.0
2019-06-25 14:13:25 TRACE ControllerService:164 - Entering withdraw method with parameters: balance = 50.0, withdraw = 0.0
2019-06-25 14:13:25 TRACE ControllerService:166 - Getting connection with user parameters: TEST_USER, p4ssw0rd
2019-06-25 14:13:25 INFO  ControllerService:172 - REMOVE BEFORE FINALIZATION! Amount to be updated to: 50.0
2019-06-25 17:25:22 TRACE ControllerService:24 - Instantiation has been restricted.  REMOVE THIS LINE BEFORE FINALIZATION!
2019-06-25 17:25:22 TRACE ControllerService:208 - Entering register method
2019-06-25 17:25:55 TRACE ControllerService:208 - Entering register method
2019-06-25 17:26:45 TRACE ControllerService:208 - Entering register method
2019-06-25 17:27:21 TRACE ControllerService:219 - Getting connection with admin parameters
2019-06-25 17:27:21 INFO  ControllerService:230 - Creating user account with parameters: TEST_USER2, password
2019-06-25 17:27:21 ERROR ControllerService:247 - Could not create user account.
java.sql.SQLSyntaxErrorException: ORA-01935: missing user or role name

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:58)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1075)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3820)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:3897)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at com.revature.service.ControllerService.register(ControllerService.java:233)
	at com.revature.controller.Controller.<init>(Controller.java:57)
	at com.revature.Main.main(Main.java:17)
2019-06-25 17:32:44 TRACE ControllerService:24 - Instantiation has been restricted.  REMOVE THIS LINE BEFORE FINALIZATION!
2019-06-25 17:32:44 TRACE ControllerService:208 - Entering register method
2019-06-25 17:33:03 TRACE ControllerService:219 - Getting connection with admin parameters
2019-06-25 17:33:03 INFO  ControllerService:230 - Creating user account with parameters: TEST_USER2, password
2019-06-25 17:33:03 ERROR ControllerService:247 - Could not create user account.
java.sql.SQLSyntaxErrorException: ORA-01935: missing user or role name

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:58)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1075)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3820)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:3897)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at com.revature.service.ControllerService.register(ControllerService.java:233)
	at com.revature.controller.Controller.<init>(Controller.java:57)
	at com.revature.Main.main(Main.java:17)
2019-06-25 17:35:32 TRACE ControllerService:24 - Instantiation has been restricted.  REMOVE THIS LINE BEFORE FINALIZATION!
2019-06-25 17:35:32 TRACE ControllerService:208 - Entering register method
2019-06-25 17:36:58 TRACE ControllerService:219 - Getting connection with admin parameters
2019-06-25 17:37:33 INFO  ControllerService:230 - Creating user account with parameters: TEST_USER2, password
2019-06-26 06:04:36 TRACE ControllerService:24 - Instantiation has been restricted.  REMOVE THIS LINE BEFORE FINALIZATION!
2019-06-26 06:04:36 TRACE ControllerService:208 - Entering register method
2019-06-26 06:04:49 TRACE ControllerService:219 - Getting connection with admin parameters
2019-06-26 06:04:51 INFO  ControllerService:245 - Creating user account with parameters: TEST_USER2, password
2019-06-26 06:04:51 INFO  ControllerService:252 - Granting admin options to created account (for purpose of testing)
2019-06-26 06:04:52 INFO  ControllerService:265 - Creating test bank account table
2019-06-26 06:09:09 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER, p4ssw0rd
2019-06-26 06:09:36 TRACE ControllerService:116 - Entering deposit method with parameters: balance = 50.0, deposit = 50.0
2019-06-26 06:09:36 TRACE ControllerService:118 - Getting connection with user parameters: TEST_USER, p4ssw0rd
2019-06-26 06:09:36 INFO  ControllerService:124 - REMOVE BEFORE FINALIZATION! Amount to be updated to: 100.0
2019-06-26 06:11:35 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER2, password
2019-06-26 06:21:05 TRACE ControllerService:24 - Instantiation has been restricted.  REMOVE THIS LINE BEFORE FINALIZATION!
2019-06-26 06:21:17 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER, p4ssw0rd
2019-06-26 06:21:30 TRACE ControllerService:208 - Entering register method
2019-06-26 06:21:42 TRACE ControllerService:219 - Getting connection with admin parameters
2019-06-26 06:21:42 INFO  ControllerService:225 - Creating user account with parameters: TEST_USER2, password
2019-06-26 06:21:44 INFO  ControllerService:232 - Granting admin options to created account (for purpose of testing)
2019-06-26 06:21:45 INFO  ControllerService:246 - Creating test bank account table
2019-06-26 06:22:27 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER2, password
2019-06-26 06:28:34 TRACE ControllerService:24 - Instantiation has been restricted.  REMOVE THIS LINE BEFORE FINALIZATION!
2019-06-26 06:28:56 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER, p4ssw0rd
2019-06-26 06:30:09 TRACE ControllerService:208 - Entering register method
2019-06-26 06:31:17 TRACE ControllerService:219 - Getting connection with admin parameters
2019-06-26 06:31:18 INFO  ControllerService:225 - Creating user account with parameters: TEST_USER2, password
2019-06-26 06:31:21 INFO  ControllerService:232 - Granting admin options to created account (for purpose of testing)
2019-06-26 06:31:22 INFO  ControllerService:246 - Creating test bank account table
2019-06-26 06:32:06 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER2, password
2019-06-26 06:32:26 TRACE ControllerService:116 - Entering deposit method with parameters: balance = 0.0, deposit = 15.0
2019-06-26 06:32:26 TRACE ControllerService:118 - Getting connection with user parameters: TEST_USER2, password
2019-06-26 06:32:26 INFO  ControllerService:124 - REMOVE BEFORE FINALIZATION! Amount to be updated to: 15.0
2019-06-26 10:54:38 TRACE ControllerService:24 - Instantiation of ControllerService has been restricted.
2019-06-26 10:54:45 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER2, password
2019-06-26 10:54:47 ERROR ControllerService:56 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:17)
	at com.revature.service.ControllerService.login(ControllerService.java:49)
	at com.revature.controller.Controller.<init>(Controller.java:39)
	at com.revature.Main.main(Main.java:17)
2019-06-26 10:55:02 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER, p4ssw0rd
2019-06-26 10:55:07 TRACE ControllerService:83 - Getting connection with user parameters: TEST_USER, p4ssw0rd
2019-07-02 02:36:48 INFO  ConnectionUtil:35 - Connection succesful
2019-07-02 06:01:05 TRACE Service:30 - Instantiation of Service class has been restricted.
2019-07-02 06:01:05 TRACE Service:67 - Searching for employee with login parameters: TestE, password
2019-07-02 06:01:05 TRACE EmployeeRepositoryJdbc:150 - Entering finding all employees
2019-07-02 06:01:06 ERROR EmployeeRepositoryJdbc:181 - Could not find all employees.
java.sql.SQLSyntaxErrorException: ORA-00942: table or view does not exist

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:58)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForDescribe(T4CPreparedStatement.java:776)
	at oracle.jdbc.driver.OracleStatement.executeMaybeDescribe(OracleStatement.java:897)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1034)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3820)
	at oracle.jdbc.driver.OraclePreparedStatement.executeQuery(OraclePreparedStatement.java:3867)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeQuery(OraclePreparedStatementWrapper.java:1502)
	at com.revature.repository.EmployeeRepositoryJdbc.findAll(EmployeeRepositoryJdbc.java:157)
	at com.revature.service.Service.login(Service.java:69)
	at com.revature.service.Service.main(Service.java:272)
2019-07-02 06:03:42 TRACE Service:30 - Instantiation of Service class has been restricted.
2019-07-02 06:03:42 TRACE Service:67 - Searching for employee with login parameters: TestE, password
2019-07-02 06:03:42 TRACE EmployeeRepositoryJdbc:150 - Entering finding all employees
2019-07-02 06:03:43 ERROR EmployeeRepositoryJdbc:181 - Could not find all employees.
java.sql.SQLSyntaxErrorException: ORA-00942: table or view does not exist

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:58)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForDescribe(T4CPreparedStatement.java:776)
	at oracle.jdbc.driver.OracleStatement.executeMaybeDescribe(OracleStatement.java:897)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1034)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3820)
	at oracle.jdbc.driver.OraclePreparedStatement.executeQuery(OraclePreparedStatement.java:3867)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeQuery(OraclePreparedStatementWrapper.java:1502)
	at com.revature.repository.EmployeeRepositoryJdbc.findAll(EmployeeRepositoryJdbc.java:157)
	at com.revature.service.Service.login(Service.java:69)
	at com.revature.service.Service.main(Service.java:272)
2019-07-02 06:06:00 INFO  ConnectionUtil:35 - Connection successful
2019-07-02 06:06:15 TRACE Service:30 - Instantiation of Service class has been restricted.
2019-07-02 06:06:15 TRACE Service:67 - Searching for employee with login parameters: TestE, password
2019-07-02 06:06:15 TRACE EmployeeRepositoryJdbc:150 - Entering finding all employees
2019-07-02 06:06:17 TRACE Service:76 - Employee not found.
2019-07-02 06:06:17 TRACE Service:67 - Searching for employee with login parameters: TestE, password
2019-07-02 06:06:17 TRACE EmployeeRepositoryJdbc:150 - Entering finding all employees
2019-07-02 06:06:18 TRACE Service:76 - Employee not found.
2019-07-02 06:13:10 TRACE EmployeeRepositoryJdbc:150 - Entering finding all employees
2019-07-02 06:17:23 TRACE EmployeeRepositoryJdbc:150 - Entering finding all employees
2019-07-02 06:20:00 TRACE EmployeeRepositoryJdbc:150 - Entering finding all employees
2019-07-02 06:20:49 TRACE Service:30 - Instantiation of Service class has been restricted.
2019-07-02 06:20:49 TRACE Service:67 - Searching for employee with login parameters: TestE, password
2019-07-02 06:20:49 TRACE EmployeeRepositoryJdbc:150 - Entering finding all employees
2019-07-02 06:20:50 TRACE Service:72 - Employee found.
2019-07-02 06:20:50 TRACE Service:67 - Searching for employee with login parameters: TestE, password
2019-07-02 06:20:50 TRACE EmployeeRepositoryJdbc:150 - Entering finding all employees
2019-07-02 06:20:51 TRACE Service:72 - Employee found.
2019-07-02 06:20:51 INFO  Service:124 - Submitting request parameters employee ID: 1, request body: This is a test submission
2019-07-02 06:20:51 TRACE RequestRepositoryJdbc:31 - Entering create method with parameter: Request [requestId=0, employee=Employee [employeeId=1, firstName=Test, lastName=Employee, username=TestE, password=password, title=Title [titleId=1, titleName=EMPLOYEE]], requestBody=This is a test submission, status=null]
2019-07-02 06:22:32 TRACE Service:30 - Instantiation of Service class has been restricted.
2019-07-02 06:22:32 TRACE Service:67 - Searching for employee with login parameters: TestM, p4ssw0rd
2019-07-02 06:22:32 TRACE EmployeeRepositoryJdbc:150 - Entering finding all employees
2019-07-02 06:22:33 TRACE Service:72 - Employee found.
2019-07-02 06:22:33 TRACE Service:67 - Searching for employee with login parameters: TestM, p4ssw0rd
2019-07-02 06:22:33 TRACE EmployeeRepositoryJdbc:150 - Entering finding all employees
2019-07-02 06:22:33 TRACE Service:72 - Employee found.
2019-07-02 06:22:33 INFO  Service:124 - Submitting request parameters employee ID: 2, request body: This is a test submission
2019-07-02 06:22:33 TRACE RequestRepositoryJdbc:31 - Entering create method with parameter: Request [requestId=0, employee=Employee [employeeId=2, firstName=Test, lastName=Manager, username=TestM, password=p4ssw0rd, title=Title [titleId=2, titleName=MANAGER]], requestBody=This is a test submission, status=null]
2019-07-02 12:33:48 TRACE Service:30 - Instantiation of Service class has been restricted.
2019-07-02 12:33:48 TRACE Service:67 - Searching for employee with login parameters: TestM, p4ssw0rd
2019-07-02 12:33:48 TRACE EmployeeRepositoryJdbc:150 - Entering finding all employees
2019-07-02 12:33:50 TRACE Service:72 - Employee found.
2019-07-02 12:33:50 TRACE Service:145 - Calling request with parameter employee ID: 2
2019-07-02 12:33:50 TRACE RequestRepositoryJdbc:95 - Entering find request by employee ID method with parameter: 2
2019-07-02 12:36:01 TRACE Service:30 - Instantiation of Service class has been restricted.
2019-07-02 12:36:01 TRACE Service:67 - Searching for employee with login parameters: TestE, p4ssw0rd
2019-07-02 12:36:01 TRACE EmployeeRepositoryJdbc:150 - Entering finding all employees
2019-07-02 12:36:03 TRACE Service:76 - Employee not found.
2019-07-02 12:44:58 TRACE Service:30 - Instantiation of Service class has been restricted.
2019-07-02 12:44:58 TRACE Service:67 - Searching for employee with login parameters: TestE, password
2019-07-02 12:44:58 TRACE EmployeeRepositoryJdbc:150 - Entering finding all employees
2019-07-02 12:44:59 TRACE Service:72 - Employee found.
2019-07-02 12:44:59 TRACE Service:145 - Calling request with parameter employee ID: 1
2019-07-02 12:44:59 TRACE RequestRepositoryJdbc:95 - Entering find request by employee ID method with parameter: 1
2019-07-02 13:00:03 TRACE Service:30 - Instantiation of Service class has been restricted.
2019-07-02 13:00:03 TRACE Service:67 - Searching for employee with login parameters: TestE, password
2019-07-02 13:00:03 TRACE EmployeeRepositoryJdbc:150 - Entering finding all employees
2019-07-02 13:00:05 TRACE Service:72 - Employee found.
2019-07-02 13:00:05 TRACE EmployeeRepositoryJdbc:64 - Entering find employee by employee ID method with parameter: 1
2019-07-03 04:33:41 TRACE Service:19 - Instantiation of Service class has been restricted.
2019-07-03 04:33:41 TRACE Service:56 - Searching for employee with login parameters: TestE, password
2019-07-03 04:33:41 TRACE EmployeeRepositoryJdbc:150 - Entering finding all employees
2019-07-03 04:33:43 TRACE Service:61 - Employee found.
2019-07-03 04:33:43 TRACE Service:150 - Viewing employee info with parameter mployee ID: 1
2019-07-03 04:33:43 TRACE EmployeeRepositoryJdbc:64 - Entering find employee by employee ID method with parameter: 1
2019-07-03 04:33:43 TRACE Service:163 - Updating employee info with parameter employee object: Employee [employeeId=1, firstName=Changed, lastName=Employee, username=Changed, password=password, title=Title [titleId=1, titleName=EMPLOYEE]]
2019-07-03 04:33:43 TRACE EmployeeRepositoryJdbc:187 - Entering update employee method
2019-07-03 04:33:43 ERROR EmployeeRepositoryJdbc:205 - Could not update employee.
java.sql.SQLSyntaxErrorException: ORA-00933: SQL command not properly ended

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:58)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1075)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3820)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:3897)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at com.revature.repository.EmployeeRepositoryJdbc.update(EmployeeRepositoryJdbc.java:201)
	at com.revature.service.Service.updateInfo(Service.java:165)
	at com.revature.service.Service.main(Service.java:174)
2019-07-03 04:34:52 TRACE Service:19 - Instantiation of Service class has been restricted.
2019-07-03 04:34:52 TRACE Service:56 - Searching for employee with login parameters: TestE, password
2019-07-03 04:34:52 TRACE EmployeeRepositoryJdbc:150 - Entering finding all employees
2019-07-03 04:34:53 TRACE Service:61 - Employee found.
2019-07-03 04:34:53 TRACE Service:150 - Viewing employee info with parameter mployee ID: 1
2019-07-03 04:34:53 TRACE EmployeeRepositoryJdbc:64 - Entering find employee by employee ID method with parameter: 1
2019-07-03 04:34:54 TRACE Service:163 - Updating employee info with parameter employee object: Employee [employeeId=1, firstName=Changed, lastName=Employee, username=Changed, password=password, title=Title [titleId=1, titleName=EMPLOYEE]]
2019-07-03 04:34:54 TRACE EmployeeRepositoryJdbc:187 - Entering update employee method
2019-07-03 04:38:24 TRACE Service:19 - Instantiation of Service class has been restricted.
2019-07-03 04:38:24 TRACE Service:56 - Searching for employee with login parameters: TestE, password
2019-07-03 04:38:24 TRACE EmployeeRepositoryJdbc:150 - Entering finding all employees
2019-07-03 04:38:25 TRACE Service:65 - Employee not found.
2019-07-03 04:39:25 TRACE Service:19 - Instantiation of Service class has been restricted.
2019-07-03 04:39:25 TRACE Service:56 - Searching for employee with login parameters: Changed, password
2019-07-03 04:39:25 TRACE EmployeeRepositoryJdbc:150 - Entering finding all employees
2019-07-03 04:39:26 TRACE Service:61 - Employee found.
2019-07-03 04:39:26 TRACE Service:150 - Viewing employee info with parameter employee ID: 1
2019-07-03 04:39:26 TRACE EmployeeRepositoryJdbc:64 - Entering find employee by employee ID method with parameter: 1
2019-07-03 04:39:26 TRACE Service:163 - Updating employee info with parameter employee object: Employee [employeeId=1, firstName=Test, lastName=Employee, username=TestE, password=password, title=Title [titleId=1, titleName=EMPLOYEE]]
2019-07-03 04:39:26 TRACE EmployeeRepositoryJdbc:187 - Entering update employee method
2019-07-03 05:07:02 TRACE Service:19 - Instantiation of Service class has been restricted.
2019-07-03 05:12:45 TRACE Service:20 - Instantiation of Service class has been restricted.
2019-07-03 05:12:45 TRACE Service:156 - Updating request info with parameter request object: Request [requestId=3, employee=null, requestBody=, status=Status [statusId=2, statusName=null]]
2019-07-03 05:12:45 TRACE RequestRepositoryJdbc:174 - Entering update request method
2019-07-03 05:12:46 INFO  RequestRepositoryJdbc:186 - Update successful!
2019-07-03 05:17:06 TRACE EmployeeRepositoryJdbc:150 - Entering finding all employees
2019-07-03 05:17:27 TRACE Service:20 - Instantiation of Service class has been restricted.
2019-07-03 05:17:27 TRACE Service:156 - Updating request info with parameter request object: Request [requestId=3, employee=null, requestBody=, status=Status [statusId=2, statusName=APPROVED]]
2019-07-03 05:17:27 TRACE RequestRepositoryJdbc:174 - Entering update request method
2019-07-03 05:17:28 INFO  RequestRepositoryJdbc:186 - Update successful!
2019-07-03 05:29:01 TRACE Service:20 - Instantiation of Service class has been restricted.
2019-07-03 05:29:01 TRACE Service:57 - Searching for employee with login parameters: TestE, password
2019-07-03 05:29:01 TRACE EmployeeRepositoryJdbc:150 - Entering finding all employees
2019-07-03 05:29:02 TRACE Service:62 - Employee found.
2019-07-03 05:29:02 TRACE Service:155 - Calling all requests.
2019-07-03 05:29:02 TRACE RequestRepositoryJdbc:136 - Entering finding all requests
2019-07-03 11:06:14 TRACE Service:20 - Instantiation of Service class has been restricted.
2019-07-03 11:06:14 TRACE Service:210 - Calling all employees.
2019-07-03 11:06:14 TRACE EmployeeRepositoryJdbc:150 - Entering finding all employees
2019-07-07 17:49:25 TRACE Service:20 - Instantiation of Service class has been restricted.
2019-07-07 17:49:25 TRACE Service:216 - Calling all employees.
2019-07-07 17:49:25 TRACE EmployeeRepositoryJdbc:150 - Entering finding all employees
2019-07-08 12:44:32 TRACE Service:20 - Instantiation of Service class has been restricted.
2019-07-08 12:44:32 TRACE Service:216 - Calling all employees.
2019-07-08 12:44:32 TRACE EmployeeRepositoryJdbc:150 - Entering finding all employees
2019-07-08 12:45:34 TRACE Service:20 - Instantiation of Service class has been restricted.
2019-07-08 12:45:34 TRACE Service:216 - Calling all employees.
2019-07-08 12:45:34 TRACE EmployeeRepositoryJdbc:150 - Entering finding all employees
2019-07-12 09:03:06 TRACE Service:23 - Instantiation of Service class has been restricted.
